app/src/main/java/com/example/recipeopedia/activities/EditProfileActivity.java

```diff
@@ -0,0 +1,101 @@
+package com.example.recipeopedia.activities;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.util.Patterns;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.Toast;
+
+import com.example.recipeopedia.R;
+import com.parse.ParseUser;
+
+public class EditProfileActivity extends AppCompatActivity {
+    public static final String TAG = "EditProfileActivity";
+    private EditText etFirstName, etLastName, etEmail, etPhoneNumber, etBio;
+    private Button btnUpdate;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_edit_profile);
+
+        etFirstName = findViewById(R.id.etFirstName);
+        etLastName = findViewById(R.id.etLastName);
+        etEmail = findViewById(R.id.etEmail);
+        etPhoneNumber = findViewById(R.id.etPhoneNumber);
+        etBio = findViewById(R.id.etBio);
+        btnUpdate = findViewById(R.id.btnUpdate);
+        btnUpdate.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                validateInput();
```

Comment:
Technically you don't need this since you call validateInput() in your editProfile() method